test/e2e/type-script-tracing.spec.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+
+import { extractWebpackErrors } from './driver/webpack-errors-extractor';
+
+describe('TypeScript Tracing', () => {
+  it.each([
+    { build: false, typescript: '~4.3.0' },
+    { build: true, typescript: '~4.3.0' },
+    { build: false, typescript: '~4.4.0' },
+    { build: true, typescript: '~4.4.0' },
+    { build: false, typescript: '~4.5.0' },
+    { build: true, typescript: '~4.5.0' },
```

Comment:
I appreciate that you add all these cases, but each case takes time so we have to keep some balance here. Otherwise, our test job will take hours :) I suggest keeping cases only for the min and max versions. I would assume that if it works for these versions, it will work for versions in-between :)